certbot-nginx/certbot_nginx/_internal/configurator.py

```diff
@@ -769,9 +769,6 @@ def enhance(self, domain, enhancement, options=None):
         except (KeyError, ValueError):
             raise errors.PluginError(
                 "Unsupported enhancement: {0}".format(enhancement))
-        except errors.PluginError:
```

Comment:
To explain why I cut this, it seems strictly redundant with [the logging already in `client.py`](https://github.com/certbot/certbot/blob/b150c9a2adaba72a61c44dbf9452596a5bb75492/certbot/certbot/_internal/client.py#L624-L625).